### ISSUE TITLE
How do I make a badge that can come into or leave existence?

### ISSUE THREAD

**Issue by aaronjensen** [trello/api-docs#80]
At the moment, it seems that there is a way to allow a badge to update, but there doesn't appear to be a way to specify that a card may eventually have a badge but does not yet.

One can return `{refresh: 10}` from a `dynamic:` callback, but that still renders an empty badge. 

Our use case is that badges can appear due to external changes (it so happens that those changes are in other cards, but it seems like it'd be nice to be able make a badge appear or disappear based on anything). 

In other words, we'd like it if the card-badges function was polled at an interval rather than only dynamic badges being polled. At the moment, we're working around this by forcing a refresh with `t.set` every 10 seconds (along with some locking to ensure we don't call that while one of our power-up iframes is open). It's a gross hack, but it seems to be the only way we can get it to work.

**Comment by bentleycook** [comment]
You're correct that there isn't a way to handle this at the moment aside from a workaround. This is the second time we've had this request come through recently so we'll talk and see if we want to re-prioritize it. I'll keep this issue updated.

**Comment by bentleycook** [comment]
I believe that this use-case will be solved when we update custom fields (a project that is underway right now). We plan to make the custom fields API a bit more robust so that you can have full CRUD access. At that point in time, you _should_ be able to POST/PUT and have the badge show only when a valid field/value exists.
